composer.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name"       : "hoa/cli",
+    "description": "The Hoa\\Cli library.",
+    "type"       : "library",
+    "keywords"   : ["library", "hoa", "console", "cli"],
+    "homepage"   : "http://hoa-project.net/",
+    "license"    : "BSD-3-Clause",
+    "authors"    : [
+        {
+            "name" : "Ivan Enderlin",
+            "email": "ivan.enderlin@hoa-project.net"
+        },
+        {
+            "name"    : "Hoa community",
+            "homepage": "http://hoa-project.net/"
+        }
+    ],
+    "support": {
+        "email" : "support@lists.hoa-project.net",
+        "irc"   : "irc://irc.freenode.org/hoaproject",
+        "source": "http://git.hoa-project.net/"
+    },
+    "require": {
+        "hoa/core"      : "~1.0 || ~2.0",
```

Comment:
Use just the highest master compatibility number. So `"hoa/core": "~2.0"` for instance.